content/influxdb/cloud/migrate-regions.md

```diff
@@ -0,0 +1,104 @@
+
+If you want to migrate data from your current InfluxDB Cloud account to your new,
+destination InfluxDB Cloud account, there is documentation available that
+walks through the migration.
+The specific process varies depending on whether your destination account is
+powered by our current database engine (Time-Structured Merge Tree or TSM) or
+[our new database engine, InfluxDB IOx](/blog/announcing-general-availability-new-database-engine/).
+
+To see what which database engine your organization is using, log into your 
+InfluxDB Cloud account and, on your organization's the homepage, look at the
+bottom of the far-right column.
+
+- [Migrate data from TSM to TSM](/influxdb/cloud/migrate-data/migrate-cloud-to-cloud/). 
+- [Migrate data TSM to IOx](/influxdb/cloud-iox/write-data/migrate-data/migrate-tsm-to-iox/)
+
+{{% note %}}
+#### Dual write into both organizations
+
+Depending on the duration of your retention policy for storing data it may be
+easier to temporarily dual write into both the source and destination accounts
+for an overlapping period until the destination account holds all desired data.
+{{% /note %}}
+
+## Migrate Resources
+
+Resources include, but are not limited to the following:
+
+- buckets
+- dashboards
+- notification rules
+- tasks
+- etc.
+
+If you have resources that you want to migrate to your destination account
+(rather than recreating these resources in the destination account) you can take
+the following steps.
+
+1.  [Download and install the 2.x `influx` CLI](/influxdb/cloud/tools/influx-cli/).
+
+2.  [Set up InfluxDB connection configurations](/influxdb/cloud/tools/influx-cli/#provide-required-authentication-credentials)
+    for both your **source** and **destination** InfluxDB Cloud accounts.
+
+    Provide the following for each connection:
+
+    - Connection configuration name
+    - [InfluxDB Cloud region URL](/influxdb/cloud/reference/regions/)
+    - [InfluxDB organization name](/influxdb/cloud/organizations/)
+    - [InfluxDB API token](/influxdb/cloud/security/tokens/)
+
+    ```sh
+    # Create your source connection configuration and set it to active
+    $ influx config create \
+      --config-name source \
+      --host-url https://cloud2.influxdata.com \
+      --org <your-source-org> \
+      --token <your-source-auth-token> \
+      --active
+
+    # Create your destination connection configuration
+    $ influx config create \
+      --config-name destination \
+      --host-url https://cloud2.influxdata.com \
+      --org <your-destination-org> \
+      --token <your-destination-auth-token>
+    ```
+
+3.  Use the [`influx export all` command](/influxdb/cloud/reference/cli/influx/export/all/#export-all-resources-in-an-organization-as-a-template)
+    to export all resources from your **source account** to an
+    [InfluxDB template](/influxdb/cloud/influxdb-templates/use/).
+
+    ```sh
+    influx export all
+    ```
+
+4.  Switch to your **destination** connection configuration.
```

Comment:
is this all in the same command line? how am I switching? by typing this command?